README.md

```diff
@@ -52,7 +52,7 @@ You can refer to the following articles on **_basics of Git and Github and also
 | 7\.      | Discovering Subdomains     | Subdomain scanner in Python using requests library                                 | [Take Me](https://github.com/dscciem/Pentesting-and-Hacking-Scripts/tree/master/Discovering%20Subdomains) |
 | 8\.      |Domino Effect    | Domino effect exploits an IBM Domino Database Security Bypass vulnerability                                 | [Take Me](/Domino%20Effect) |
 | 9\.      |TargetLib    | Library for generating network ranges, random ips and randomizing iplists.                                 | [Take Me](/TargetLib) |
-
+| 10\.     |mac spoof    | python script to spoof mac address in linux.                                 | [Take Me](https://github.com/himanshu007-creator/Pentesting-and-Hacking-Scripts/tree/himanshu007-creator-patch-1/mac%20spoof) |
```

Comment:
Just Keep this changes rest Don't make any changes in the `README`

Reply:
Done sir.